"net/http.lock

```diff
@@ -0,0 +1,147 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
I'm not sure why this file is called `"net/http.lock` and not `Gopkg.lock`. Can you delete this file and run `dep ensure` again?

Reply:
yes. I have been thinking about how this too!